manifests/integrations/activemq_xml.pp

```diff
@@ -0,0 +1,92 @@
+# Class: datadog_agent::integrations::activemq_xml
+#
+# This class will install the necessary configuration for the activemq_xml integration
+#
+# Parameters:
+#   $url
+#       URL to gather the activemq_xml starts from
```

Comment:
Maybe mention this should point to ActiveMQ web admin's URL, and not to the service itself.